prime-router/src/main/kotlin/azure/observability/event/OrderFacilitySummary.kt

```diff
@@ -0,0 +1,27 @@
+package gov.cdc.prime.router.azure.observability.event
+
+import org.hl7.fhir.r4.model.Base
+import org.hl7.fhir.r4.model.Organization
+
+data class OrderingFacilitySummary(
+    val orderingFacilityName: String = UNKNOWN,
+    val orderingFacilityState: String = UNKNOWN,
+) {
+    companion object {
+        const val UNKNOWN = "Unknown"
+
+        /**
+         * Create an instance of [OrderingFacilitySummary] from a [Organization]
+         */
+        fun fromOrganization(requester: Base): OrderingFacilitySummary {
```

Comment:
We should have unit tests for these methods with all the different logic going on, especially PerformerSummary.